run-test.py

```diff
@@ -95,8 +95,6 @@
             - NET_ADMIN
       ports:
         - containerPort: 22
-  nodeSelector:
-    jepsen: "true"
```

Comment:
why do we need to delete it? 
 it needs to restrict the node. you know that very well.

Reply:
For the record - after discussion we figured out it's OK.